pages/common/irb.md

```diff
@@ -0,0 +1,11 @@
+# irb
+
+> Interactive Ruby Shell.
+
+- Run Ruby Sell with load some gem:
+
+`irb -r {{net/http}}`
+
+- Don't show result:
```

Comment:
What is the use of this option ? Is this used commonly enough ? I don't see a usecase where you launch a REPL shell but do not want to see command output.